public/apps/configuration/panels/permission-list/edit-modal.tsx

```diff
@@ -39,6 +39,7 @@ interface PermissionEditModalDeps {
   groupName: string;
   action: Action;
   allowedActions: string[];
+  dataSourceId: string;
```

Comment:
Probably not necessary, I don't think you can just read it when defining the function?

Reply:
yep, removed.